en-US/Docs/ReleaseNotesInsiderPreview.md

```diff
@@ -22,158 +23,192 @@ You can review linked terms by pasting the forward link into your browser window
 
 ## What's new in this build: 
 * Updated OS files including core OS bug fixes.
-* Issues with Windows Device Portal/WEBB not showing apps, processes and issues with Bluetooth pairing have been addressed.
-* Issues with speech recognition on the Raspberry Pi have been addressed.
-* Issues with the display on an MBM not displaying after a disconnect/reconnect event have been addressed.
-* An issue with passwords for default accounts resetting during updates when using the provided FFUs has been addressed.
 
 ## Known issues in this build:
-* The Raspberry Pi3 built-in Bluetooth driver currently only supports low bandwidth devices.
-* Pin.Read is failing when using GPIO on the MBM with the DMAP/Lightning Providers due to issues in the Lightning Nuget.
-* In some cases, the mouse pointer is not visible after deploying or debugging apps with Visual Studio.
-* When using the SoftAP clients will not be able to access content exposed by UAP apps. See Server Application with SoftAP below for details.
+* Remote Display may fail with the Dragon Board.
+
+## IoT Core general known issues and work arounds
```

Comment:
correct heading level?  this heading doesn't make sense without content underneath it